libs/common-utilities/src/constants/contracts.ts

```diff
@@ -36,3 +36,38 @@ export const CONTRACTS: ContractAddressList = {
     '0x5': '0x000000000000cd17345801aa8147b8d3950260ff',
   },
 };
+
```

Comment:
see unused import on this one.

Reply:
Are you referring to the first line of the file? I can remove that

Reply:
I removed the unused import above